util-linux/pkg.yaml

```diff
@@ -5,10 +5,10 @@ dependencies:
   - stage: base
 steps:
   - sources:
-      - url: https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.37/util-linux-2.37.tar.xz
```

Comment:
Is there a reason to use one host or another here, or is this merely copy-paste?